tests/hlsl-intrinsic/shader-execution-reordering/hit-object-reorder-thread.slang

```diff
@@ -74,14 +76,14 @@ void rayGenerationMain()
         someValues);
 
     uint r = calcValue(hit);
-    // SPIRV: OpReorderThreadWithHitObjectNV
```

Comment:
Why is this test removed?

Reply:
I broke the tests during development; I forgot to add them back since I broke them at the start; I am adding them 👍